onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -1543,16 +1547,21 @@ def aten_empty_strided(size: INT64, stride: INT64) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_eq(self: TensorType, other: TensorType) -> TensorType:
+@torch_op("aten::eq")
+def aten_eq(self: TTensor, other: TTensor) -> BOOL:
     # eq.Tensor(Tensor self, Tensor other) -> Tensor
 
-    raise NotImplementedError()
+    return op.Equal(self, other)  # type: ignore[arg-type]
```

Comment:
Remove   # type: ignore[arg-type] here and everywhere else?

Reply:
got it